[SYCL][Doc] Mention PI_CUDA as a possible SYCL BE envvar value (#1286)
intel/llvm: sycl/doc/EnvironmentVariables.md

| Environment variable | Values | Description |
| -------------------- | ------ | ----------- |
| SYCL_PI_TRACE | Any(\*) | Force tracing of PI calls to stderr. |

> | SYCL_BE | PI_OPENCL, PI_CUDA, PI_OTHER | When SYCL RT is built with PI, this controls which plugin to use. Default value is PI_OPENCL. |

Reviewer: ```suggestion
| SYCL_BE | PI_OPENCL, PI_CUDA, PI_OTHER | When SYCL RT is built with PI, this controls which plugin is used by the `default_selector`. Default value is PI_OPENCL. |
```

Reviewer: As far as I understand all plugins are still available irrespective of this variable, and one can pick any device (OpenCL or CUDA) with a dedicated device selector; only the default device selector is affected by `SYCL_BE`.

Author: Thanks for noting this, I've applied the suggestion
